integration-tests/sample-site/src/main/webapp/sanity/modifiers/wait-for-element-to-be-visible/failed.jsp

```diff
@@ -0,0 +1,30 @@
+<%--
+
```

Comment:
Instead of having two pages with different time for element to show after,
we could have only one (called `wait-for-me.jsp`) and then configure different timeout for our success and failure suites.

Or am I missing something?